packages/gatsby-page-utils/package.json

```diff
@@ -27,11 +27,12 @@
     "gatsby-core-utils": "^1.3.5",
     "glob": "^7.1.6",
     "lodash": "^4.17.15",
-    "micromatch": "^3.1.10"
```

Comment:
Did you intentionally major-bump micromatch? I've seen a few reported issues related to micromatch bumping recently so I'd prefer not to change this if at all possible.

Reply:
There is a problem with the types for 3.1.10. 
Updated so it only uses the types from 4.0.1 now

Reply:
Yeah that's super. Thanks! :)